drivers/sony/displays/bravia.cr

```diff
@@ -0,0 +1,269 @@
+require "placeos-driver/interface/powerable"
+require "placeos-driver/interface/muteable"
+require "placeos-driver/interface/switchable"
+
+class Sony::Displays::Bravia < PlaceOS::Driver
+  include Interface::Powerable
+  include Interface::Muteable
+
+  INDICATOR = "\x2A\x53" # *S
+  HASH      = "################"
+  ERROR     = 70
+
+  # Discovery Information
+  tcp_port 20060
+  descriptive_name "Sony Bravia LCD Display"
+  generic_name :Display
+
+  enum Inputs
+    Tv1
+    Tv2
+    Tv3
+    Hdmi1
+    Hdmi2
+    Hdmi3
+    Mirror1
+    Mirror2
+    Mirror3
+    Vga1
+    Vga2
+    Vga3
+
+    def index : Int32
+      to_s.gsub(/[^0-9]/, "").to_i
+    end
+
+    def type_hint : String
+      case self
+      when Hdmi1, Hdmi2, Hdmi3
+        "1000"
+      when Mirror1, Mirror2, Mirror3
+        "5000"
+      when Vga1, Vga2, Vga3
+        "6000"
+      else
+        "0000"
+      end
```

Comment:
When we have the potential for exhaustive case via `in`, it's preferred over `when`